File: official/docs/java/brand/UpdateBrand.java
import com.easypost.EasyPost;

public class UpdateBrand {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        User user = User.retrieveMe();

        Map<String, Object> params = new HashMap<>();
        params.put("color", "303F9F");

        Brand brand = user.updateBrand(params);

        System.out.println(brand);
    }
}

File: official/docs/go/brand/update_brand.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	user, _ := client.RetrieveMe()
	brand, _ := client.UpdateBrand(
		map[string]interface{}{
			"color": "#303F9F",
		},
		user.ID,
	)

	fmt.Println(brand)
}
